src/app.js

```diff
@@ -5,13 +5,33 @@ import cors from 'cors';
 import logger from 'morgan';
 import passport from 'passport';
 import dotenv from 'dotenv';
+import socketIO from 'socket.io';
 import session from 'express-session';
 import './utilities/socialAuthStrategies';
 import routes from './routes';
 
 // Create global app object
 const app = express();
 
+// let's start our server...
+const server = app.listen(process.env.PORT || 4001, () => {
+  console.log(`Server is up and running on port ${server.address().port}...!`); //eslint-disable-line
+});
+
+const io = socketIO(server);
+
+io.on('connection', (socket) => {
+  socket.on('inAppNotifications', (data) => {
+    socket.emit('inAppNotifications', data);
+  });
+});
+
```

Comment:
I think the setup for socket.io, see line 23, should be in its own file and imported into the app.js file so that in the nearest future when we need to add more configurations or need to implement other features on the app that needs socket.io to be re-configured, we will work in that file and not in the app.js file. This will make our code more modular.

Reply:
You are probably right but I will leave it as it is for now.  It's definitely something to consider in any subsequent refactoring.